src/public/utils.ts

```diff
@@ -38,7 +38,14 @@ function stopWaiting() {
  * Go back to linkurious home page
  */
 function goToLinkurious(sourceKey: string) {
-  const location = window.location.origin + `/dashboard?key=${sourceKey}`;
+  let basePath;
+  const match = window.location.pathname.match(/([^/]+)\/plugins\/.+/);
+  if (match === null) {
+    basePath = '';
+  } else {
+    basePath = match[1];
+  }
+  const location = window.location.origin + basePath + `/dashboard?key=${sourceKey}`;
```

Comment:
`window.location.origin + basePath` is wrong when basePath is defined, it would look like `linkurio.us.combasePath`